src/main/java/teammates/logic/api/GateKeeper.java

```diff
@@ -327,21 +327,27 @@ public void verifyAccessibleForCurrentUserAsInstructorOrTeamMember(AccountAttrib
             String section, String email) {
         InstructorAttributes instructor = instructorsLogic.getInstructorForGoogleId(courseId, account.googleId);
         if (instructor != null) {
-            if (!instructor.isAllowedForPrivilege(section,
-                    Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS)) {
-                throw new UnauthorizedAccessException("Instructor does not have enough privileges to view the photo");
-            }
+            verifyInstructorCanViewPhoto(instructor, section);
             return;
         }
-
```

Comment:
Unrelated change; revert this (also, the blank line helps to separate blocks of code).